src/main/java/mate/academy/springboot/datajpa/service/CategoryService.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.springboot.datajpa.service;
+
+import mate.academy.springboot.datajpa.model.Category;
+
+public interface CategoryService {
+    Category save(Category category);
+
+    Category getById(Long id);
```

Comment:
let's be consistent
choose one word `find` or `get` and then use it everywhere, do not mix it up